data/sql/updates/pending_db_world/rev_1676604656261986500.sql

```diff
@@ -0,0 +1,117 @@
+--
+DELETE from `creature` where `guid`=77831 AND `id1`=22060;
+INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(77831, 22060, 0, 0, 530, 0, 0, 1, 1, 1, -2024.52, -10786.1, 81.9137, 0.664982, 3600, 0, 0, 751, 0, 2, 0, 0, 0, '', 0);
+
+UPDATE `creature_template_addon` SET `bytes1` = 0 WHERE (`entry` = 22060);
+
+-- Pathing for Entry: 22060
+SET @NPC := 77831;
+SET @path := @NPC * 10;
+UPDATE creature SET wander_distance=0,MovementType=2,position_x=-1970.9095,position_y=-10545.881,position_z=176.9451 WHERE guid=@NPC;
+DELETE FROM creature_addon WHERE guid=@NPC;
+INSERT INTO creature_addon (guid,path_id,mount,bytes1,bytes2,emote,visibilityDistanceType,auras) VALUES (@NPC,@path,0,0,1,0,0, '34189');
+DELETE FROM waypoint_data WHERE id=@path;
+INSERT INTO waypoint_data (id,point,position_x,position_y,position_z,orientation,delay,move_type,action,action_chance,wpguid) VALUES
```

Comment:
backticks

Reply:
Discord ate the backticks. I don't know how I didn't notice it